<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoOrthoSliceCtrl</class>
 <widget class="QDialog" name="SoOrthoSliceCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoOrthoSlice control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="interpolationLabel">
     <property name="text">
      <string>interpolation</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="alphaUseLabel">
     <property name="text">
      <string>alphaUse</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="alphaUseCombo">
     <item>
      <property name="text">
       <string>ALPHA_AS_IS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALPHA_OPAQUE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALPHA_BINARY</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="clippingSideCombo">
     <item>
      <property name="text">
       <string>FRONT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BACK</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="sliceNumberSlider">
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="clippingSideLabel">
     <property name="text">
      <string>clippingSide</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="axisCombo">
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="sliceNumberEdit">
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="interpolationCombo">
     <item>
      <property name="text">
       <string>NEAREST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LINEAR</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="clippingCheckBox">
     <property name="text">
      <string>clipping</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="axisLabel">
     <property name="text">
      <string>axis</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="sliceNumberLabel">
     <property name="text">
      <string>sliceNumber</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
